plugins/source/aws/resources/services/docdb/cluster_parameters.go

```diff
@@ -20,46 +19,29 @@ func clusterParameters() *schema.Table {
 		Resolver:    fetchDocdbClusterParameters,
 		Transform:   transformers.TransformWithStruct(&types.Parameter{}),
 		Columns: []schema.Column{
-			client.DefaultAccountIDColumn(false),
-			client.DefaultRegionColumn(false),
+			client.DefaultAccountIDColumn(true),
+			client.DefaultRegionColumn(true),
+			{
```

Comment:
Why not add `arn` to parent just like we have in rds/neptune?

Reply:
The parent here is engine version that doesn't have ARN, as defined by the [docs](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazondocumentdbelasticclusters.html#amazondocumentdbelasticclusters-resources-for-iam-policies)